### PR TITLE
Rick-stache Mask

### PR DESCRIPTION
**Pull request by tommysalami3**
## About The Pull Request

Fun little mask I made that you can buy from autodrobe or win from arcades. Mostly just a way for me to learn some DM.

## Why It's Good For The Game

More drip, can be used to make fun of me, and goes across your nose so you look really funny.

## Changelog

add: Adds a comically small orange moustache cosmetic to autodrobes and arcades.

/:cl:

### CODE REVIEW

**Comment by StoneMason** [comment]
Having it be a wig like thing (changing between the hair types) would be pretty cool for an upstream PR.

**Comment by FernandoJ8** [comment]
Add pics or it didn't happen.

**Comment by tommysalami3** [comment]
HIDEFACIALHAIR so it does work as a wig for facial hair.

**Comment by tommysalami3** [comment]
image
comically small....

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -186,3 +186,10 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/rickstache
```
Comment (JohnFulpWillard) [inline code/modules/clothing/masks/miscellaneous.dm:190]: You should make this in its own file

**Comment by tommysalami3** [comment]
im closing this and finishing it later because its become insanely fucked up from just changing the file locations what the fuck

**Comment by SgtHunk** [comment]
As a project itself, this was fine. But imo this is pointless bloat in the chance you want it merged. All this does is add an obscure moustache that references a regular? added by the regular himself?
I'm against this due to this not adding anything  valuable, if we don't have a contributing objective set in stone.

**Comment by tommysalami3** [comment]
"Mostly just a way for me to learn some DM"

Just wanted something to put through the proper channels and work on so I can learn while talking to willard about how shit works. If I wanted to seriously add this i would've bundled it in with alot more items added to the autodrobe that fit the theme and done it after getting a grip on how dm works.

**Comment by SgtHunk** [comment]
It was a good attempt, would've checked out on a codebase that isn't required to avoid upstream files
That commit history tho

**Comment by tommysalami3** [comment]
i didnt realize i could f5 without committing until far too late

**Pull request closed without merging**